src/about.rs

```diff
@@ -12,12 +12,11 @@ pub fn present_window(transient_for: Option<&impl IsA<gtk::Window>>) {
     let win = adw::AboutWindow::builder()
         .modal(true)
         .application_icon(APP_ID)
-        .application_name(gettext("Mousai"))
-        .developer_name(gettext("Dave Patrick Caberto"))
+        .application_name("Mousai")
+        .developer_name("Dave Patrick Caberto")
         .version(VERSION)
-        .copyright(gettext("© 2023 Dave Patrick Caberto"))
+        .copyright("© 2023 Dave Patrick Caberto")
         .license_type(gtk::License::Gpl30)
-        // Translators: Replace "translator-credits" with your names. Put a comma between.
```

Comment:
I'm not sure why is this removed.

Reply:
Like i said! Is superfluous, as these comments are already in the Weblate instance with the respective strings

Reply:
> these comments are already in the Weblate instance with the respective strings

Not sure if I missed some Weblate feature, but it is only displayed from that comment.

![image](https://github.com/SeaDve/Mousai/assets/64297935/e0886b56-1e46-46a0-9901-e1f72968028b)

Reply:
Please keep the comment of traslator-credits. They are visible on translation tools and applications.

![resim](https://github.com/SeaDve/Mousai/assets/6218679/5380b628-f8a3-4d11-bcf0-28a37b6cb709)